pkg/virtctl/root.go

```diff
@@ -18,10 +19,20 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/vnc"
 )
 
+var programName string
+
 func NewVirtctlCommand() *cobra.Command {
+	programName = "virtctl"
+	if filepath.Base(os.Args[0]) == "kubectl-virt" {
```

Comment:
maybe add a comment that this is the case when being used a krew plugin

Reply:
Actually that's a good point, noone will know why that's needed... Except the ones who read the git comments